src/mmw/js/src/modeling/modificationConfigUtils.js

```diff
@@ -2,6 +2,15 @@
 
 var modificationConfig = require('./modificationConfig.json');
 
+// Useful for testing.
+function setConfig(config) {
```

Comment:
Consider renaming this and `resetConfig` to `setModificationConfig` and `resetModificationConfig` to be more representative. Currently we're using "settings" for runtime configuration in `clientSettings` but I can imagine `setConfig` being confusing down the line.